### PR TITLE
Team page refactor

### PR DESCRIPTION
**Pull request by leoRysing**
# What was the ticket?

Quick run through to set a code pattern and clean up the teams / team-managment page.

# What did I do?
- Restructured the file structure to move things out of the components
- Added a (partial) new hook to allow pages to auto resize and update to mobile view (has to be combined with the query for now because legacy code doesn't use it yet)
- Created the new management page, with a reusable footer component for use in the future team pages
- Added a mobile version of the teams landing page
- Cleaned up the footers, to also auto work together with resizing

# How did I test it?

Describe in detail steps you used to test the changes you have made.

- Running locally, resizing frequently, and using the chrome device emulator to match with legacy code mobile versioning
- Going through the flows on this page

Required checks:

- [x] Did you conduct a self-review?
- [ ] Have you written unit or integration tests?

# What could go wrong in the future? What parts of your code should the reviewer pay the most attention to?

- Some styling could get messed up, but I believe it should be cleared up for now

Describe aspects of the PR that may become problems in the future.

# Additional Comments for the Reviewers

# Screenshots

FIGMA

alt text

MY VERSION

alt text

### CODE REVIEW

File: src/shared/useWebsite.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    const handleResize = () => {
+        if (window.innerWidth > 650) {
+            setIsWebsite(true);
+        } else {
+            setIsWebsite(false);
+        }
+    };
+
+    useEffect(() => {
+            console.log("is it website?", isWebsite);
```
Comment (fraander) [inline src/shared/useWebsite.js:17]: Probably should comment this out/remove it

File: src/pages/TeamsPage/ExpandedTeams/style.css [hunk 2]
```diff
@@ -11,7 +11,7 @@
 }
 
 .paragraph-text {
-    font-family: "Space Mono";
```
Comment (fraander) [inline src/pages/TeamsPage/ExpandedTeams/style.css:14]: Should we be setting text styles more globally??

File: src/pages/TeamsPage/ExpandedTeams/CommonTeam/TeamPageFooter.jsx [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+const pages = [
+  { name: "Generate", link: "/" },
+  { name: "Apply", link: "/apply" },
+  { name: "About", link: "/about" },
+  { name: "Culture", link: "/culture" },
+  { name: "Teams", link: "/teams" },
+  // { name: "People", link: "/", disabled: true },
+  { name: "Projects", link: "/projects" },
+];
+const pages2 = [
```
Comment (fraander) [inline src/pages/TeamsPage/ExpandedTeams/CommonTeam/TeamPageFooter.jsx:19]: this naming seems like a red-flag

File: src/data/teamDataV2.js [hunk 4]
```diff
@@ -16,34 +16,34 @@ import SoftwareLargePicture from "../assets/images/TeamsPageImages/TeamsPagePH3.
 import HardwareLargePicture from "../assets/images/TeamsPageImages/TeamsPagePH4.png"
 import EngagementLargePicture from "../assets/images/TeamsPageImages/TeamsPagePH5.png"
 
-const teamDataV2 = [
-    {
+// yellow - "#FFBF3C"
+const teamDataV2 = [{
```
Comment (fraander) [inline src/data/teamDataV2.js:20]: This naming seems like an anti-pattern

File: src/component/VerticalFooter/index.jsx [hunk 5]
```diff
@@ -1,53 +1,53 @@
...
-          </div>
-        </div>
-      </Row>
-      <Row className="footer-text"> <a href="/about">About </a></Row>
-      <Row className="footer-text"> <a href="/culture">Culture</a></Row>
-      <Row className="footer-text"> <a href="/">Teams</a></Row>
-      <Row className="footer-text"> <a href="/">People</a></Row>
-      <Row className="footer-text"> <a href="/apply">Apply</a></Row>
-      <Row className="footer-text"> <a href="/">Projects</a></Row>
+// export default function VerticalFooter() {
```
Comment (fraander) [inline src/component/VerticalFooter/index.jsx:13]: Delete?

File: src/component/HorizontalFooter/index.jsx [hunk 6]
```diff
@@ -1,104 +1,104 @@
...
+//         <div className="logo-background sherm">
+//            <a href="https://generatenu.com/">
+//             <img src={GenerateLogo} height="50vh" width="50vw" alt="Logo description" />
+//           </a>
+//         </div>
+//       </div>
+//       </Row>
+//       <Row className="footer-text">
+//         {" "}
+//         <a href="/about">About </a>
```
Comment (fraander) [inline src/component/HorizontalFooter/index.jsx:27]: Delete?

File: src/component/HorizontalFooter/index.js [hunk 7]
```diff
@@ -1,110 +1,140 @@
...
 import InstaIcon from "../../assets/images/socialMediaIcons/Insta.png";
 import LinkedInIcon from "../../assets/images/socialMediaIcons/Linkedin.png";
 import FbIcon from "../../assets/images/socialMediaIcons/Facebook.png";
 import YoutubeIcon from "../../assets/images/socialMediaIcons/Youtube.png";
 import MosaicIcon from "../../assets/images/socialMediaIcons/Mosaic.png";
 import NortheasternIcon from "../../assets/images/socialMediaIcons/Northeaster.png";
 import Sherm from "../../assets/images/socialMediaIcons/Sherm.png";
+import { useNavigate } from "react-router-dom";
+
+const pages = [
```
Comment (fraander) [inline src/component/HorizontalFooter/index.js:14]: Aren't these somewhere else as well? Move to /shared?

**Review by fraander — DISMISSED** [review]

File: src/pages/TeamsPage/ExpandedTeams/style.css (hunk 2 above)
Comment (leoRysing) [reply src/pages/TeamsPage/ExpandedTeams/style.css:14]: It is set, just some styles reset it here and there. I can remove it but it's a carry over from legacy code

File: src/data/teamDataV2.js (hunk 4 above)
Comment (leoRysing) [reply src/data/teamDataV2.js:20]: Legacy code, going to rename it

File: src/component/HorizontalFooter/index.jsx (hunk 6 above)
Comment (leoRysing) [reply src/component/HorizontalFooter/index.jsx:27]: index.jsx (from what I can tell isn't used, will delete. For some reason there's two files in this folder?

File: src/component/HorizontalFooter/index.js (hunk 7 above)
Comment (leoRysing) [reply src/component/HorizontalFooter/index.js:14]: The icons? Assets is a partial shared

**Review by fraander — DISMISSED** [review]

**Review by fraander — APPROVED** [review]

**Pull request merged**